examples/bevy/slint_bevy_adapter.rs

```diff
@@ -178,6 +178,7 @@ pub async fn run_bevy_app_with_slint(
         app.set_runner(runner);
         app.insert_resource(BackBuffer(None));
         bevy_app_pre_default_plugins_callback(&mut app);
+        app.insert_resource(ClearColor(Color::NONE));
```

Comment:
I don’t think that this should be the default. IMO this belongs into main and should be at the discretion of the developer, while we stick to bevy‘s default.

Reply:
To clarify, this is where IMO it should go:

https://github.com/slint-ui/slint/blob/a176a553522adb052aa7c50b7e23ef2f9f3ce604/examples/bevy/main.rs#L100